test_conformance/extensions/cl_khr_command_buffer/negative_command_buffer_create.cpp

```diff
@@ -289,8 +289,9 @@ struct CreateCommandBufferDeviceDoesNotSupportOutOfOderQueue
     {
         BasicCommandBufferTest::Skip();
```

Comment:
I'm not sure if this is related to the issue you are fixing, but we're ignoring the return value from this call, and I'm pretty sure it should beshould be
```cpp
        if (BasicCommandBufferTest::Skip()) return true;
```

This can be a separate GitHub issue/PR to fix if it's unrelated to your issue though (might be other occurrences we need to fix as part of that).

Reply:
I agree that should be fixed. But I think it is not related to the issue I'm fixing here.